src/build/functions/edge.ts

```diff
@@ -132,9 +132,11 @@ const copyHandlerDependencies = async (
 
   const outputFile = join(destDir, `server/${name}.js`)
 
-  // Prepare environment variables for draft-mode (i.e. __NEXT_PREVIEW_MODE_ID, __NEXT_PREVIEW_MODE_SIGNING_KEY, __NEXT_PREVIEW_MODE_ENCRYPTION_KEY)
-  for (const [key, value] of Object.entries(env)) {
-    parts.push(`process.env.${key} = '${value}';`)
+  if (env) {
+    // Prepare environment variables for draft-mode (i.e. __NEXT_PREVIEW_MODE_ID, __NEXT_PREVIEW_MODE_SIGNING_KEY, __NEXT_PREVIEW_MODE_ENCRYPTION_KEY)
+    for (const [key, value] of Object.entries(env)) {
+      parts.push(`process.env.${key} = '${value}';`)
+    }
```

Comment:
https://github.com/opennextjs/opennextjs-netlify/actions/runs/15105688788/job/42453967192#step:14:1785

next@^13 did not have `env` defined (it was added in https://github.com/vercel/next.js/commit/dcff078936c72f43d2a2e37a7899b1a6f631855a ), so we need to check if it's defined first